de.hamstersimulator.objectsfirst.inspector/src/main/java/de/hamstersimulator/objectsfirst/inspector/ui/InputControl.java

```diff
@@ -260,6 +416,12 @@ private void onMultiTypeChanged(final Type newType) {
         }
     }
 
+    /**
+     * Creates a simple input control
+     * This control can handle all primitives, their wrappers, enums, and subtypes of object.
+     * It cannot handle object or optional on its own, as these require an input type dropdown.
```

Comment:
Inconsistency in the documentation. Sometimes Object and Optional are written with capital O and sometimes not

Reply:
changed all to uppercase with d5ae83d9b6d00633c06a658d67118784234e9597